### PR TITLE
Add missing translations for Swedish and English locales

### PR DESCRIPTION
**Pull request by kustaahuhtala**
#### Summary

Add missing translations for Swedish and English versions.

Screenshot 2020-12-21 at 14 17 31
Screenshot 2020-12-21 at 14 17 26
Screenshot 2020-12-21 at 14 17 20

### CODE REVIEW

File: frontend/packages/enduser-frontend/src/localization/app/en.json [hunk 1]
```diff
@@ -639,6 +639,9 @@
           "description": "Finnish and Swedish pre-primary education begins on 11 August 2021.",
           "description2": "<br/>If you need early childhood education starting from 1 August 2021 before the start of pre-primary education, you can apply for it with this application by selecting “I also apply for early childhood education related to pre-primary education”.",
           "label": "Start date",
+          "instructions": {
+            "text": "You can postpone your desired start date until the processing of your application starts. After this, you can make changes to your desired start date by contacting pre-primary education service counselling (tel. 09 816 31000)."
```
Comment (mikkopiu) [inline frontend/packages/enduser-frontend/src/localization/app/en.json:643]: Do we have any way of knowing when to update this or should we rather link to an external reference (espoo.fi)?

File: frontend/packages/enduser-frontend/src/localization/app/sv.json [hunk 2]
```diff
@@ -639,6 +639,9 @@
           "description": "Förskolan börjar i 11.8.2021.",
           "description2": "<br>Om ditt barn behöver småbarnspedagogisk verksamhet fr.o.m. 1.8.2021 före förskolan börjar kan du ansöka om sådan med denna blankett. Välj ”Jag ansöker om småbarnspedagogik i samband med förskoleundervisning”.",
           "label": "Inledningsdatum",
+          "instructions": {
+            "text": "Du kan senarelägga önskad startdatum så länge ansökan har inte tagits till behandling . När ansökan har är under behandling vänligen ta kontakt med servicenummer 09 81627600 när du vill ändra startdatum."
```
Comment (mikkopiu) [inline frontend/packages/enduser-frontend/src/localization/app/sv.json:643]: Extra space:
```suggestion
            "text": "Du kan senarelägga önskad startdatum så länge ansökan har inte tagits till behandling. När ansökan har är under behandling vänligen ta kontakt med servicenummer 09 81627600 när du vill ändra startdatum."
```

and I assume the number is the Swedish number and therefore different to the above?

File: frontend/packages/enduser-frontend/src/localization/app/en.json (hunk 1 above)
Comment (kustaahuhtala) [reply frontend/packages/enduser-frontend/src/localization/app/en.json:643]: There is no exact date for a single application in terms of processing. In any case, most of the applicants will apply either on the first day of the pre-primary education or even earlier.

File: frontend/packages/enduser-frontend/src/localization/app/sv.json (hunk 2 above)
Comment (kustaahuhtala) [reply frontend/packages/enduser-frontend/src/localization/app/sv.json:643]: Yes, it's the Swedish number. I will remove the extra whitespace.

File: frontend/packages/enduser-frontend/src/localization/app/en.json (hunk 1 above)
Comment (mikkopiu) [reply frontend/packages/enduser-frontend/src/localization/app/en.json:643]: Sry no, I meant the phone number :D

Comment (kustaahuhtala) [reply frontend/packages/enduser-frontend/src/localization/app/en.json:643]: I suggest we leave it as it is. Customer seems to be very familiar with the contents of the application and adding extra step by linking to espoo.fi would (imo) make it just more confusing.

**Review by mikkopiu — APPROVED** [review]

**Pull request merged**